application.js

```diff
@@ -8,43 +8,49 @@
 function runChallenges() {
 
   // Ex 1. Read the content of the email input
-  var email = null;  // TODO: replace null, keep the email variable.
+    var email = "dleuliette@gmail.com";  // TODO: replace null, keep the email variable.
 
 
   // Ex 2. Fill the content of the email input with your email
-
+    $("#email").val("elodieandrieux62100@gmail.com");
 
 
   // Ex 3. Replace the email hint (next to the input) with 'This is my email now'
   //       The text should be emphasized using a <strong> tag
-
+    $("#email-hint").html('<strong>This is my email now</strong>');
```

Comment:
Be carefull to __don't mix__ singe quote & double quotes

    $('#email-hint').html('<strong>This is my email now</strong>');

:rocket: